src/python/espressomd/shapes.py

```diff
@@ -266,3 +269,73 @@ class HollowConicalFrustum(Shape, ScriptInterfaceHelper):
     .. image:: figures/conical_frustum.png
     """
     _so_name = "Shapes::HollowConicalFrustum"
+
+
+@script_interface_register
+class ShapeUnion(Shape, ScriptObjectRegistry):
```

Comment:
I think this should be called `Union`, it's clear from the context that this is about `Shapes`.